FCFS.cpp

```diff
@@ -0,0 +1,73 @@
+// C++ program for implementation of FCFS

```

Comment:
https://www.geeksforgeeks.org/program-for-fcfs-cpu-scheduling-set-1/
Plagiarism